set.mm

```diff
@@ -537490,14 +537496,36 @@ singleton on a couple (with disjoint domain) at a point in the domain
     ZUTUMUOVAGUOVAULVAUOULUNEBNOPQACRUAUMUOUBUKUTUOUPUKUTUQUKUTHUOUPABCUCUDTUEU
     FTUGQUKUPUOSUMUKUPUOABCUHTUIUJ $.
 
+  ${
+    bj-isvec.scal $e |- ( ph -> K = ( Scalar ` V ) ) $.
+    $( The predicate "is a vector space".  (Contributed by BJ, 6-Jan-2024.) $)
+    bj-isvec $p |- ( ph -> ( V e. LVec <-> ( V e. LMod /\ K e. DivRing ) ) ) $=
```

Comment:
I removed the `l` from the label since I think there is no reason to specify "left" for modules or vector spaces : we just say when we define them that our modules are left modules, and then that's it.  I would like the tokens to be changed to `Vec` and `Mod` instead of `LVec` and `LMod`, by the way.

I removed the `d` from the label because I thought that for more advanced theorems (i.e., past the elementary parts of logic and set theory), deduction form was the "default" form, and had no suffix in the label ? But @tirix and you have proved more advanced theorems of that form, so you probably know what would be best.

Reply:
> I removed the `d` from the label because I thought that for more advanced theorems (i.e., past the elementary parts of logic and set theory), deduction form was the "default" form, and had no suffix in the label ? But @tirix and you have proved more advanced theorems of that form, so you probably know what would be best.

In general you are right (if this would be the only theorem of this kind), but in this case, we already have ~islvec in set.mm, so after your proposed renamings and moving ~bj-isvec (with removing the prefix "bj-") to main, we would have a clash of labels...

Reply:
> But @tirix and you have proved more advanced theorems of that form, so you probably know what would be best.

Yes, in general I skip the `d` in more advanced theorems, as most of them would have it, making the distinction pointless.